src/reducers/account/index.js

```diff
@@ -20,11 +20,19 @@ const initialState = {
    sentMessage: false
 }
 
-const loaderReducer = (state, { ready }) => {
+const loaderReducerPendingStack = []
+
+const loaderReducer = (state, { type, ready }) => {
    if (typeof ready === 'undefined') {
       return state
    }
-   return { ...state, formLoader: !ready }
+
+   if (!ready) {
+      loaderReducerPendingStack.push(type)
```

Comment:
@marcinbodnar what is the plan for `type`?  Looks like for now just using a counter would be enough as `.pop()` return value is never used.

Reply:
@vgrichina I thought it will be good to know what exactly is happening there with actions, it might also be useful in the future to know what actions are pending. But it can be easily replaced with a counter. Want me to change it?

Reply:
@marcinbodnar more like I wanted to understand what is the plan. 

> I thought it will be good to know what exactly is happening there with actions, it might also be useful in the future to know what actions are pending.

if we want to have it inspectable, shouldn't it be part of Redux state then so that it's visible in Redux dev tools?

Reply:
@vgrichina Sure, good idea. Done.